tests/proc/lif/test_process.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+import unittest
+import numpy as np
+from lava.proc.lif.process import LIF
+
+
+class TestLIFProcess(unittest.TestCase):
+    """Tests for LIF class"""
+    def test_init(self):
+        """Tests instantiation of LIF"""
+        lif = LIF(shape=(100,),
+                  du=100*np.ones((100,), dtype=np.float),
+                  dv=np.ones((100,), dtype=np.float),
+                  bias=2*np.ones((100,), dtype=np.float),
+                  bias_exp=np.ones((100,), dtype=np.float),
+                  vth=np.ones((100,), dtype=np.float))
+
+        self.assertEqual(lif.shape, (100,))
+        self.assertListEqual(lif.du.init.tolist(), 100 * [100.])
```

Comment:
I guess we should update the get() method to return the init value when called before compilation.

Reply:
So this is not relevant to this code then.